src/uri/ersip_transport.erl

```diff
@@ -107,6 +107,8 @@ is_datagram({transport, tls}) ->
     false;
 is_datagram({transport, tcp}) ->
     false;
+is_datagram({transport, sctp}) ->
+    false;
```

Comment:
I thought that SCTP is message-oriented transport so we do not need stream parser there.

Reply:
Oh.. I misunderstood `is_datagram`. Of course, SCTP is message-oriented transport.
Maybe rename `is_datagram` to `is_message_oriented`? I can do this. What do you think?